eng/common/scripts/Service-Level-Readme-Automation.ps1

```diff
@@ -42,51 +42,64 @@ param(
 
 Set-StrictMode -Version 3
 
-function create-metadata-table($readmeFolder, $readmeName, $moniker, $msService, $clientTableLink, $mgmtTableLink, $serviceName)
+# Parse out the rest content of service readme after excluding index table and metadata.
+function Get-RestContent($readmePath) {
+  $restContent = ''
+  if (Test-Path $readmePath) {
+    $readmeContent = Get-Content $readmePath -Raw
```

Comment:
It is kind of difficult to understand what this is doing. Can you please write some tests so we can see that expected input and output to ensure it is doing what we expect.